appsettings.json

```diff
@@ -1,6 +1,7 @@
 {
   "ConnectionStrings": {
-    "AzureSQL": "KEY"
+    "AzureSQL": "KEY",
+    "TestSQL": "Server=localhost;Database=DotNetCourseDatabase;Trusted_Connection=false;TrustServerCertificate=True;User Id=sa;Password=SQLConnect1!;"
```

Comment:
Nawet devowego ConnectionStringa nie powinieneś przesyłać, stworzyłem "AzureSQL": "KEY" po to żebyś mógł sobie lokalnie podstawić tam klucz połączenia i go nie pushować, a pipeline będzie tam wklejał klucz połączenia dynamicznie z sekretu